File: UserServer/src/main/java/cc/nobrain/dev/userserver/domain/alarm/service/dto/AlarmReadDto.java
package cc.nobrain.dev.userserver.domain.alarm.service.dto;

import cc.nobrain.dev.userserver.domain.base.dto.BaseDto;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Data;
import lombok.experimental.Accessors;

import java.io.Serializable;
import java.time.OffsetDateTime;

/**
 * DTO for {@link cc.nobrain.dev.userserver.domain.alarm.entity.AlarmRead}
 */
@Data
@Accessors(chain = true)
@JsonIgnoreProperties(ignoreUnknown = true)
public class AlarmReadDto extends BaseDto implements Serializable {
    private Long id;
    private boolean readYn;
    private boolean deleteYn;
    private OffsetDateTime readAt;
    private OffsetDateTime deleteAt;
}

File: UserServer/src/main/kotlin/cc/nobrain/dev/userserver/domain/alarm/service/dto/AlarmReadDto.kt
package cc.nobrain.dev.userserver.domain.alarm.service.dto

import java.io.Serializable
import java.time.OffsetDateTime

/**
 * DTO for {@link cc.nobrain.dev.userserver.domain.alarm.entity.AlarmRead}
 */
open class AlarmReadDto(
    var id: Long? = null,
    var readYn: Boolean? = null,
    var deleteYn: Boolean? = null,
    var readAt: OffsetDateTime? = null,
    var deleteAt: OffsetDateTime? = null
) : Serializable